spec/models/admin_user_spec.rb

```diff
@@ -0,0 +1,22 @@
+describe AdminUser do
+  it 'return type should be User' do
+    testuser = AdminUser.find_or_create
+    expect(testuser).to be_a_kind_of(User)
+  end
+  context 'no existing admin' do
+    it 'first_name and last_name should equal to the specified values' do
```

Comment:
Again, the labels are meant to form human phrases, with "it" referring to the method. So for example, `it 'returns a user...'`

Also: "specified value" is vague. You can instead use "default value".

Finally, split the two expect statements into separate tests.